js/views/chatview.js

```diff
@@ -160,8 +161,8 @@
 				relativeDate = moment(timestamp, 'x').diff(moment()) > -86400000;
 
 			var actorDisplayName = commentModel.get('actorDisplayName');
-			if (commentModel.attributes.actorType === 'guests') {
-				// FIXME get guest name from WebRTC or something like that
+			if (commentModel.get('actorType') === 'guests' &&
+				actorDisplayName === null) {
```

Comment:
this works, but existing comments are not updated, when a user changes their name